index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ali Ahmad Karimi Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+    
+</head>
+
+<body>
+    <section>
+        <nav class="toolbar">
+            <h3 class="logo">Ali Ahmad Karimi</h3>
+            <img src="images/Icon-Menu.png" alt="Menu Icon">
+        </nav>
+        <div class="container">
+            <div class="welcome">
+                <h2>I'm Ali Ahmad Karimi Glad to see you!</h2>
+                <p class="details">I’m a software developer! I can help you build a product , feature or website Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+            </div>
+            <div class="social">
+                <p class="connect">LET’S CONNECT</p>
+                <img src="images/Twitter.png" alt="Twitter">
+                <img src="images/Linkedin-icon.png" alt="Linkedin" class="sIcon">
+                <img src="images/Github.png" alt="Github" class="sIcon">
+            </div>
+        </div>
+    </section>
```

Comment:
Please note the following: 😁😁

- [ ] As mentioned in the other comment about the header, this section should be separated from the header it should be in a `section` tag on its own, kindly change that.
- [ ] The `h2` tag you have should be an `h1` as it serves as your main title, kindly change that.
- [ ] The list of icons should be a `ul` element and you'll need to use `li` for every icon you have, and each icon should be a link which means you'll need to use an `a` tag, please change that.

After fixing all these, you should have a better-constructed HTML that is easy to read! 😎